pkg/helper/validate/validate.go

```diff
@@ -15,15 +15,26 @@
 package validate
 
 import (
-	k8s "k8s.io/client-go/kubernetes"
+	"fmt"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s "k8s.io/client-go/kubernetes"
 )
 
+type params interface {
```

Comment:
Interface to get kube client and namespace

Reply:
:heart_eyes_cat: